doc/STYLE.md

```diff
@@ -13,6 +13,7 @@ $ cd kovri/ && cpplint src/path/to/my/file && [edit file manually to apply fixes
 
 - Vim integration
   - [clang-format](http://clang.llvm.org/docs/ClangFormat.html#vim-integration)
+  - [clang-format ubuntu 16.04 vim workaround](http://stackoverflow.com/questions/39490082/clang-format-not-working-under-gvim)
```

Comment:
Please try not to mix completely unrelated commits with a topic PR. In the future, if you split up PR's, when one PR needs more development before merging, the other PR won't have to wait.